test/pjrt/test_runtime_tpu.py

```diff
@@ -92,6 +92,7 @@ def test_xla_devices_multiprocess(self):
     devices_per_process = pjrt.run_multiprocess(xm.xla_device)
     self.assertDictEqual(devices_per_process, expected)
 
+  @absltest.skipIf(tpu.num_available_devices() > 4, "Not implemented")
```

Comment:
Please make this `tpu.num_available_chips`. This test case is still valid when there are multiple devices per chip. Also, it's safer to do `tpu.num_available_chips() != 4` here. I don't know the behavior of `TPU_VISIBLE_CHIPS=0,1,2,3` when there are fewer than 4 chips actually available.

Reply:
Thanks, Will, updated